packages/repo/src/mst/mst.ts

```diff
@@ -96,6 +105,7 @@ export class MST implements DataStore {
     opts?: Partial<MstOpts>,
   ): Promise<MST> {
     const pointer = await util.cidForEntries(entries)
+    // TODO: why isn't this layer=null
```

Comment:
Hmmm not sure that it really matters & maybe it's just preference. 

`null` is mainly about "not lazy loaded yet" whereas I think of a new MST as being default `0` 

I'm down to change to `null` & perhaps it is actually more honest. Doesn't affect any of the logic 👍

Reply:
also a total nit but actually kinda important cause I search on it all the time, we normally do `@TODO` instead of `TODO`

Reply:
It looks like all call sites of `create()` already setting the layer explicitly, also. I felt like it should be "not lazy loaded yet" until a layer was actually computed. `layer=0` is an actual well-defined layer.

Changed to `null` and tests pass.